dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -214,20 +214,31 @@ public void testLinkageReports() {
             "106 target classes causing linkage errors referenced from 516 source classes.");
 
     Nodes dependencyPaths = details.query("//p[@class='linkage-check-dependency-paths']");
-    Node dependencyPathMessageOnProblem = dependencyPaths.get(dependencyPaths.size() - 2);
+    Node dependencyPathMessageOnProblem = dependencyPaths.get(dependencyPaths.size() - 4);
     Assert.assertEquals(
-        "The following path contains guava-jdk5-17.0.jar:",
+        "The following paths contain guava-jdk5-13.0.jar:",
```

Comment:
With the change, the version of guava-jdk5 picked up by the dashboard has changed.